app/assets/stylesheets/changes.scss

```diff
@@ -0,0 +1,8 @@
+// Place all the styles related to the Changes controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+iframe {
+	height: 800px;
+	width: 100%;
+}
```

Comment:
I think these styles should just be inline in the html. This css will be general to all iframes in the codebase (not that there are others, but it lays a trap for future developers).

Reply:
Totally fine with that. Wasn't entirely certain how the css for each controller worked. Would this css be limited to just iframes used in views connected to the changes controller? Will move this to the html and it will simplify things.

Reply:
It's not limited in that way, no. As an organizational convention Rails creates a css file for each controller, but there isn't any automatic scoping or anything.

Reply:
OK. That was my understanding. Definitely didn't want to set a global iframe style with this. Thanks for explaining!